charts/kof-mothership/templates/sveltos/kof-cluster-role-regional.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: k0rdent.mirantis.com/v1alpha1
+kind: MultiClusterService
+metadata:
+  name: kof-regional-cluster
+spec:
+  clusterSelector:
+    matchLabels:
+      k0rdent.mirantis.com/kof-cluster-role: regional
+  serviceSpec:
+    priority: 100
+    services:
+
+      - name: ingress-nginx
+        namespace: ingress-nginx
+        template: ingress-nginx-4-11-3
+
+      - name: cert-manager
+        namespace: cert-manager
+        template: cert-manager-1-16-2
+        values: |
+          cert-manager:
+            crds:
+              enabled: true
+
+      - name: kof-storage
+        namespace: kof
+        template: kof-storage-{{ .Values.kcm.kof.charts.storage.version | replace "." "-" }}
+        values: |
+          {{- /*
+            `kof-operator` should create this `MultiClusterService`
```

Comment:
We cannot create this MultiClusterService from ClusterDeployment.
As it's not clear how to deal with multiple ClusterDeployments that should be covered by MultiClusterService.

Probably we need to create a signle Service per each regional cluster.
If we need to create multiple regional clusters with a similar configuration, then we need something like a new "KofClusterClass" to inherit config.